docs/requirements.txt

```diff
@@ -1,7 +1,6 @@
-furo>=2022.6.21
-jinja2>=3.0.3, <3.1.0
+furo>=2022.12.7
 myst-nb>=0.17.1
 myst-parser>=0.18.1
-sphinx-copybutton>=0.5.0
-sphinx-toolbox>=3.1.2
+sphinx-copybutton>=0.5.1
+sphinx-toolbox>=3.2.0
```

Comment:
Bumping versions should probably be a separate PR as unrelated to dropping 3.7

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/526/commits/3e370499a7edc12328fb86034c0dbe8be725b9d4